ugc/types.hpp

```diff
@@ -341,6 +341,48 @@ struct UGC
            (m_ratings.empty() || m_totalRating <= kInvalidRatingValue || m_basedOn == 0);
   }
 
+  // Packs rating and reviews number to uint8_t.
+  // Uses first two bits for reviews number:
+  //  0 - 0 reviews
+  //  1 - 1 or 2 reviews
+  //  2 - from 2 to 10 reviews
+  //  3 - more than 10 reviews
+  // This constants may be changed if we will achive large count of reviews.
```

Comment:
s/This/These/
s/we will achive/we achieve/

Reply:
done